src/common/error/validation.ts

```
@@ -0,0 +1,7 @@
+import { DomainError } from './doamin-error';
+
+export class DomainValidationError extends DomainError {
+  constructor(message: string) {
+    super(message);
+  }
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)